src/twisted/conch/test/test_keys.py

```diff
@@ -1025,6 +1113,28 @@ def test_toOpenSSHECDSA(self):
                 keydata.publicECDSA_openssh[:-8]) # no comment
 
 
+    def test_toOpenSSHED25519(self):
+        """
+        L{keys.Key.toString} serializes a ED25519 key in OpenSSH format.
+        """
+        key = keys.Key.fromString(keydata.privateED25519openssh)
+        self.assertEqual(key.public().toString('openssh', b'comment'),
+                         keydata.publicED25519openssh)
+
+        # We need to patch the comparison value because SecureRandom
+        # gets patched when run as a unit test.
```

Comment:
How is this comparison value "patched"?  Also; can this refer more specifically to what does the `SecureRandom` patching, rather than just alluding to it?  Long-term, this is the sort of thing we should parameterize rather than patch.